cartographer/sensor/compressed_point_cloud.cc

```diff
@@ -61,8 +61,8 @@ RangefinderPoint CompressedPointCloud::ConstIterator::operator*() const {
   return {current_point_};
 }
 
-CompressedPointCloud::ConstIterator& CompressedPointCloud::ConstIterator::
-operator++() {
+CompressedPointCloud::ConstIterator&
+CompressedPointCloud::ConstIterator::operator++() {
```

Comment:
For whatever reason this was formatted by your bot.

Reply:
Ha, it happened before. Sometimes it formats not only files found in the commit, but other files in the code base.